app/src/main/java/com/example/ai_stuttering_assistant/ui/theme/Theme.kt

```diff
@@ -57,8 +58,9 @@ fun AIStutteringAssistantTheme(
     if (!view.isInEditMode) {
         SideEffect {
             val window = (view.context as Activity).window
-            window.statusBarColor = colorScheme.primary.toArgb()
-            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = darkTheme
+//            window.statusBarColor = colorScheme.primary.toArgb()
+            window.statusBarColor = Color.TRANSPARENT
+            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = !darkTheme
```

Comment:
These 2 lines make the status bar transparent and also make the status bar items (such as the time and battery indicator) either dark text or light text depending on if the system dark mode is enabled or not.